bin/next-build-tools.js

```diff
@@ -228,6 +228,7 @@ program
 	.option('--procfile', 'Runs all processes specified in the Procfile')
 	.option('-s, --script <file>', 'Runs a single javascript file')
 	.option('--subargs [subargs]', 'Sub arguments to pass to a single script', /^\[.+]$/)
+	.option('--no-nodemon', 'Do not run through nodemon')
```

Comment:
for v4 of nbt I think we should use the `--no` prefix instead of every `--skip`. The conversion is so neat